docs/source/tutorials/oregon.rst

```diff
@@ -0,0 +1,688 @@
+
+Now let's examine how Medicaid enrollment affects the distribution of emergency department visits (rather than costs):
+
+.. code-block:: python
+
+    # Initialize local estimators for visits analysis
+    simple_local_estimator = dte_adj.SimpleLocalDistributionEstimator()
+    ml_local_estimator = dte_adj.AdjustedLocalDistributionEstimator(
+        LinearRegression(),
+        folds=5
+    )
+
+    # Fit local estimators on the full dataset
+    # Parameters: X, treatment_arms, treatment_indicator, outcomes, strata
+    simple_local_estimator.fit(X, Z, D, Y_NUM_VISIT_CENS_ED, strata)
+    ml_local_estimator.fit(X, Z, D, Y_NUM_VISIT_CENS_ED, strata)
+
+    # Define evaluation points for emergency department visits
+    outcome_ed_visits_locations = np.linspace(Y_NUM_VISIT_CENS_ED.min(), Y_NUM_VISIT_CENS_ED.max(), 20)
+
+Distribution Treatment Effects: Visits Analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # Compute LDTE: Treatment vs Control
+    ldte_ctrl, lower_ctrl, upper_ctrl = simple_local_estimator.predict_ldte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=outcome_ed_visits_locations
+    )
+
+    # LDTE: Treatment vs Control
+    ldte_simple, lower_simple, upper_simple = simple_local_estimator.predict_ldte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=outcome_ed_visits_locations
+    )
+
+    ldte_ml, lower_ml, upper_ml = ml_local_estimator.predict_ldte(
+        target_treatment_arm=1,  # Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Not selected for treatment (Not enrolled)
+        locations=outcome_ed_visits_locations
+    )
+
+    # Visualize the local distribution treatment effects using dte_adj's built-in plot function
+    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(15, 6))
+
+    # Visualize Treatment vs Control using dte_adj's plot function
+    plot(outcome_ed_visits_locations, ldte_simple, lower_simple, upper_simple,
+         title="Treatment vs Control (Simple Local Estimator)",
+         xlabel="Emergency Department Visits", ylabel="Local Distribution Treatment Effect",
+         color="purple",
+         ax=ax1)
+
+    plot(outcome_ed_visits_locations, ldte_ml, lower_ml, upper_ml,
+         title="Treatment vs Control (ML-Adjusted Local Estimator)",
+         xlabel="Emergency Department Visits", ylabel="Local Distribution Treatment Effect",
+         ax=ax2)
+
+    plt.tight_layout()
+    plt.show()
+
+.. image:: ../_static/oregon_ldte_visits.png
+   :alt: Oregon Health Insurance Experiment LDTE Visits Analysis
+   :width: 800px
+   :align: center
+
+Probability Treatment Effects: Visits Analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
```

Comment:
Same for this, we can remove this code block entirely as it's duplicated with the content below